blog/2023-11-22-config-validation-and-error-handling.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Jan Bouwhuis
+authorURL: https://github.com/jbouwh
+authorImageURL: https://avatars.githubusercontent.com/u/7188918?s=96&v=4
+title: Config processing and error handling
+---
+
+## Changes in component config processing and error handling
+
+The way component YAML configuration is processed has been changed. Now, it is possible to raise if an error occurs. Some custom integrations might break if they are using `config.async_process_component_and_handle_errors`.
+Instead, they can use `config.async_process_component_and_handle_errors` now. This new method supports raising when an error occurs during config processing.
```

Comment:
The second method name is the same as the first method name.

Reply:
Updated the first method name to `config.async_process_component_config` and adjusted the publication date.